utils/src/test/resources/log4j.properties

```diff
@@ -12,3 +12,4 @@ log4j.logger.org.eclipse.jetty=ERROR
 
 # TransmogrifAI logging
 log4j.logger.com.salesforce.op=ERROR
+log4j.logger.com.salesforce.op.utils.spark.OpSparkListener=INFO
```

Comment:
you can set this `log4j.logger.com.salesforce.op.utils.spark.OpSparkListener=INFO` programmatically in OpSparkListenerTest